src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
 public class Duke {
+
+    //Zhou Zhou
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        UI console = new UI();
+        TaskManager taskManager = new TaskManager();
+        String input = UI.input();
+        while (!input.equals("bye")) {
+            String[] words = input.split(" ");
+            String arguments = input.substring(words[0].length());
+            switch (words[0]) {
+            case "help":
+                UI.response(Command.HELP);
+                break;
+            case "please":
+                UI.response(Command.PLEASE);
+                break;
+            case "list":
+                UI.response(Command.LIST);
+                break;
+            case "mark":
+                TaskManager.markAsDone(Integer.parseInt(arguments.trim()));
+                UI.response(Command.MARK, TaskManager.Tasks[Integer.parseInt(arguments.trim())].listTask());
+                break;
+            case "unmark":
+                TaskManager.markAsNotDone(Integer.parseInt(arguments.trim()));
+                UI.response(Command.UNMARK, TaskManager.Tasks[Integer.parseInt(arguments.trim())].listTask());
+                break;
+            case "todo":
+                TaskManager.addTask(Command.TODO, arguments.trim());
+                UI.response(Command.ADD, TaskManager.Tasks[TaskManager.getTasksCount()].listTask());
+                break;
+            case "event":
+                TaskManager.addTask(Command.EVENT, arguments.trim());
+                UI.response(Command.ADD, TaskManager.Tasks[TaskManager.getTasksCount()].listTask());
+                break;
+            case "deadline":
+                TaskManager.addTask(Command.DEADLINE, arguments.trim());
+                UI.response(Command.ADD, TaskManager.Tasks[TaskManager.getTasksCount()].listTask());
```

Comment:
Not too sure if this is more than 120 characters but seem quite long so be careful with such lines